### PR TITLE
Update example notebooks for signac 2.

### PR DESCRIPTION
**Pull request by bdice**
This PR updates the example notebook contents for signac 2.

I would appreciate help rewriting notebook 106 to use HOOMD 3.

### CODE REVIEW

**Comment by bdice** [comment]
Conda exited with code 137 here: https://app.circleci.com/pipelines/github/glotzerlab/signac-examples/256/workflows/b911701e-f97e-4227-9054-a65418d54358/jobs/528

That's an "out of memory" error. I think we need to overhaul the signac-examples CI to use GitHub Actions and install with mamba. However, I think that should be prioritized _after_ signac 2.0. For now, I would like to consider merging this with failing CI, since we know the notebooks can't be run until signac 2.0 is released. We need the new notebooks in our docs at the same time that signac 2.0 is released, and then we'll need to rewrite the example projects after the 2.0 release. @glotzerlab/signac-committers What do you think?

File: notebooks/signac_106_signac-flow_HOOMD-blue_Example.ipynb [hunk 1]
```diff
@@ -32,19 +32,14 @@
    "metadata": {},
```
Comment (b-butler) [inline notebooks/signac_106_signac-flow_HOOMD-blue_Example.ipynb:32]: <p>This needs to be rerun.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Review by b-butler — APPROVED** [review]
I made the changes I wanted to.

**Comment by b-butler** [comment]
> I would appreciate help rewriting notebook 106 to use HOOMD 3.

Done.

We should wait to merge until signac 2.0 is out and we can check the tests then.

**Comment by bdice** [comment]
@b-butler Thanks! I can re-render the notebooks with the new HOOMD and push again.

**Comment by bdice** [comment]
This should be fixed up now. I think it's ready to merge (CI will fail as expected).

**Pull request merged**